agent/structs/config_entry_routes.go

```diff
@@ -216,6 +217,10 @@ type HTTPHeaderFilter struct {
 	Set    map[string]string
 }
 
+type URLRewrite struct {
+	Path string
```

Comment:
I suspect this will need to become an optional `*string` in the future, but not something that has to be resolved in this PR

Reply:
I will do a wraparound